REFERENCE.md

```diff
@@ -0,0 +1,265 @@
+# Reference Guide
+
+## Configuration Specification
+
+(This text is copied from the original README.md, carrying over the notes from [our Google Doc](https://docs.google.com/document/d/1HP65B3VZDOMeSpNMYgXz73bMdpeMb0siixKs3U4JvaM/edit#))
+
+### Declarative Data Quality Configs
+
+#### Rule Bindings
+**Rule Bindings**:
+Defines a single Data Quality validation routine.
+Each value declared in `entity_id`, `column_id`, `filter_id`, and `rule_id`
+is a lookup key for the more detailed configurations that must be defined in their respective configurations files.
+```yaml
+rule_bindings:
+  T2_DQ_1_EMAIL:
+    entity_id: TEST_TABLE
+    column_id: VALUE
+    row_filter_id: DATA_TYPE_EMAIL
+    incremental_time_filter_column_id: TS
+    rule_ids:
+      - NOT_NULL_SIMPLE
+      - REGEX_VALID_EMAIL
+      - CUSTOM_SQL_LENGTH_LE_30
+    metadata:
+      team: team-2
+
+  T3_DQ_1_EMAIL_DUPLICATE:
+    entity_id: TEST_TABLE
+    column_id: VALUE
+    row_filter_id: DATA_TYPE_EMAIL
+    rule_ids:
+      - NO_DUPLICATES_IN_COLUMN_GROUPS:
+          column_names: "value"
+      - NOT_NULL_SIMPLE
+    metadata:
+      team: team-3
+```
+Each `rule_binding` must define one of the field `entity_id` or `entity_uri` and all of the fields `column_id`, `row_filter_id`, and `rule_ids`. `entity_id` refers to the table being validated, `entity_uri` is the URI path in a remote metadata registry that references the table being validated, `column_id` refers to the column in the table to be validated, `row_filter_id` refers to a filter condition to select rows in-scope for validation, and `rule_ids` refers to the list of data quality validation rules to apply on the selected column and rows.
```

Comment:
1. "one of the fields" --> typo

2. here and below - suggest to structure the parameters into a list for reading simplicity